packages/cli/package.json

```diff
@@ -159,6 +159,7 @@
     "open": "^7.0.0",
     "openapi-types": "^10.0.0",
     "p-cancelable": "^2.0.0",
+    "p-lazy": "^3.1.0",
```

Comment:
Why not the latest `4.0.0`?

Reply:
`4.0.0` (like most of the latest versions of packages from `sindresorhus`) only support ESM, and our repo is unfortunately still not fully ready to work with ESM-only packages.
Upgrading this package, or any other ESM-only package will break our tests.